sdks/java/core/src/main/java/org/apache/beam/sdk/options/ProxyInvocationHandler.java

```diff
@@ -324,12 +324,13 @@ private void populateDisplayData(DisplayData.Builder builder) {
   }
 
   /**
-   * Marker interface used when the original {@link PipelineOptions} interface is not available at
+   * Marker interface used when the original {@link PipelineOptions} interface is not known at
```

Comment:
It turns out that PipelineOptionsFactory requires each argument to map to an actual pipeline option property, it will throw an error if it doesn't. If you turn off strict parsing, then unknown and malformed args are dropped. so the from "command line" can't happen. Its only JSON deserialization that has this occur.